lib/console/console_io.rb

```diff
@@ -0,0 +1,74 @@
+require "console/game_factory"
+
+module RestartOptions
+  YES = "Y"
+  NO = "n"
+end
+
+class ConsoleIO
+  def initialize(output = $stdout, input = $stdin)
+    @out = output
```

Comment:
Ruby gives you nice syntax to do parallel assignment. `@out, @in = output, input`

Reply:
@nickbdyer is it better in any way Vs. this way? 😬

Reply:
I tried changing it, but I couldn't:
```ruby
def initialize(output, input = $stdout, $stdin)
    @out = output
    @in = input
end
```
But it looks like that sintax doesn't work for method signatures:

```
SyntaxError:
/Users/jgarcia/Repositories/Apprenticeship/TicTacToeRB/lib/console/console_io.rb:9: formal argument cannot be a global variable
(output, input = $stdout, $stdin)
```